CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unpublished
+* Add an inverse flag to the phase banner
```

Comment:
Nitpick: can you fix the lack of a line space here? You probably just copied it from below but that one is inconsistent with the rest of this file.

Reply:
⛏ 👍